cdn.py

```diff
@@ -8,8 +8,11 @@
 @roles('class-cache')
 def fastly_purge(*args):
     "Purge items from Fastly, eg \"/one,/two,/three\". Wildcards not supported."
-    govuk_fastly = 'http://www-gov-uk.map.fastly.net'
-    hostnames_to_purge = ['www.gov.uk', 'assets.digital.cabinet-office.gov.uk']
+    if env.environment == 'production':
+        hostnames_to_purge = ['www.gov.uk', 'assets.digital.cabinet-office.gov.uk']
+    elif env.environment == 'staging':
+        hostnames_to_purge = ['www.staging.alphagov.co.uk', 'assets.staging.alphagov.co.uk']
```

Comment:
I think we can use `publishing.service.gov.uk` as we'll only need to change this later, and it's rare that we ever use the Staging CDN service.